js/apps/admin-ui/cypress/support/pages/admin-ui/manage/identity_providers/social/ProviderMicrosoftGeneralSettings.ts

```diff
@@ -0,0 +1,29 @@
+import ProviderBaseGeneralSettingsPage from "../ProviderBaseGeneralSettingsPage";
+
+const tenant_id_input_test_value = "tenant_id_input_test_value";
+
+export default class ProviderMicrosoftGeneralSettings extends ProviderBaseGeneralSettingsPage {
+  private tenantIdInput = "#microsoftTenantId";
```

Comment:
If possible we prefer to use test id's to retrieve elements during test, for more information see the [Cypress Testing Library documentation](https://testing-library.com/docs/cypress-testing-library/intro/).

Reply:
Hi, not sure what you mean here - this file is pretty much copypasted from [ProviderGoogleGeneralSettings](https://github.com/keycloak/keycloak/blob/main/js/apps/admin-ui/cypress/support/pages/admin-ui/manage/identity_providers/social/ProviderGoogleGeneralSettings.ts) verbatim, and all the Provider tests use this method to get the input element.

Reply:
You can search the project for `findByTestId` to get an idea of what needs to be done. Older code still uses `id` as we haven't updated it to match the new conventions everywhere.

Reply:
Thanks, I updated it to `findByTestId`. I also see the tests are failing, but it does not seem to be related to this PR.